packages/communication-react/review/communication-react.api.md

```diff
@@ -1879,7 +1879,16 @@ export const MessageStatusIndicator: (props: MessageStatusIndicatorProps) => JSX
 
 // @public
 export interface MessageStatusIndicatorProps {
+    // (undocumented)
+    message?: ChatMessage;
```

Comment:
I guess we could do the calculation out side of MessageStatusIndicator in MessageThread, message object seems too big for the indicator